source/replication.rst

```diff
@@ -195,3 +202,82 @@ The *Replica* site will be ready for replication when you apply changes as usual
 
    $ kubectl apply -f deploy/cr.yaml
 
+.. _operator-replication-mcs:
+
+Enabling multi-cluster Services
+--------------------------------------------
+
+Kubernetes `multi-cluster Services (MCS) <https://cloud.google.com/kubernetes-engine/docs/concepts/multi-cluster-services>`_
+is a cross-cluster discovery and invocation of Services. MCS-enabled Services become discoverable and accessible across clusters with a virtual IP address.
+
+This feature allows splitting applications into multiple clusters combined in
+one *fleet*, which can be useful to separate logically standalone parts
+(i.e. stateful and stateless ones), or to address privacy and scalability
+requirements, etc.
+
+Multi-cluster Services should be supported by the cloud provider. It is
+supported `by Google Kubernetes Engine (GKE) <https://cloud.google.com/kubernetes-engine/docs/concepts/multi-cluster-services>`__, and `by Amazon Elastic Kubernetes Service (EKS) <https://aws.amazon.com/blogs/opensource/introducing-the-aws-cloud-map-multicluster-service-controller-for-k8s-for-kubernetes-multicluster-service-discovery/>`__.
+
+Configuring your cluster for multi-cluster Services includes two parts:
+
+* configure MCS with your cloud provider,
+* make needed preparations with the Operator.
+
+To set up MCS for a specific cloud provider you should follow official guides,
+for example ones `from Google Kubernetes Engine (GKE) <https://cloud.google.com/kubernetes-engine/docs/how-to/multi-cluster-services>`__, or `from Amazon Elastic Kubernetes Service (EKS) <https://aws.amazon.com/blogs/opensource/introducing-the-aws-cloud-map-multicluster-service-controller-for-k8s-for-kubernetes-multicluster-service-discovery/>`__.
+
+Setting up the Operator for MCS makes it registering a Services for export to
+other clusters `with a ServiceExport object <https://cloud.google.com/kubernetes-engine/docs/how-to/multi-cluster-services>`_.
+Set the following options in the ``multiCluster`` subsection of the ``deploy/cr.yaml``
+configuration file to make it happened:
+
+* ``multiCluster.enabled`` should be set to ``true``,
+* ``multiCluster.DNSSuffix`` string should be equal to the cluster domain suffix
+  for multi-cluster Services used by Kubernetes (``svc.clusterset.local``
+  `by default <https://cloud.google.com/kubernetes-engine/docs/how-to/multi-cluster-services>`_).
+
+The initial ServiceExport creation and sync with the clusters of the fleet takes
+approximately five minutes. After ServiceExport object is created, exported
+Services can be resolved from any Pod in any fleet cluster as
+``SERVICE_EXPORT_NAME.NAMESPACE.svc.clusterset.local``.
```

Comment:
Usually export is fast but import can take time. Could we add the commands to check import and export? 
kubectl get serviceimport
kubectl get serviceexport